pkg/test/framework/repository/paths.go

```diff
@@ -0,0 +1,37 @@
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// Package repository have utility methods related to accessing a local istio code repository. The repository
+// path is deduced from ${GOPATH} environment variable.
+package repository
```

Comment:
This file seems somewhat redundant with https://github.com/istio/istio/blob/master/pkg/test/env/istio.go.

I suggest adding the helm-specific variables there.